index.html

```diff
@@ -258,10 +258,10 @@ <h2>Contact me</h2>
                     I’d love to help with it!</p>
             </header>
 
-            <form action="" id="contact-me-form">
-                <input type="text" name="user-name">
-                <input type="text" name="email">
```

Comment:
- [ ] As requested earlier, there is still a bigger pace  ( left and right) on your form section. Kindly adjust the padding to match the original design

![image](https://user-images.githubusercontent.com/98453979/235642638-5e984954-b63f-45b3-a9a0-fdbcaf811c74.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/235643103-6f95028b-acd5-4e58-8926-7eb8c9e3da39.png)